x-pack/plugins/cloud_security_posture/public/pages/vulnerabilities/latest_vulnerabilities_table.tsx

```diff
@@ -89,9 +89,11 @@ const customCellRenderer = (rows: DataTableRecord[]) => ({
       {({ finding }) => <SeverityStatusBadge severity={finding.vulnerability.severity} />}
     </CspVulnerabilityFindingRenderer>
   ),
-  'data_stream.dataset': ({ rowIndex }: EuiDataGridCellValueElementProps) => (
+  'observer.vendor': ({ rowIndex }: EuiDataGridCellValueElementProps) => (
     <CspVulnerabilityFindingRenderer row={rows[rowIndex]}>
-      {({ finding }) => <>{getDatasetDisplayName(finding.data_stream.dataset) || '-'}</>}
+      {({ finding }) => (
+        <>{finding.observer?.vendor || getDatasetDisplayName(finding.data_stream.dataset) || '-'}</>
```

Comment:
are we missing `data_stream.dataset` fallback here?

Reply:
In my opinion for simplicity and consistency, we could use the same logic across the board. More some we don't really expect this edge case to really happen until we allow users to see any data (not only from Elastic integrations) in our flows. 

> The reason I decided to use that as a fallback in the flyouts is that those are all custom fields and not just a table.

This is true for now, but for example in the Alert flyout it's possible to filter in/out pretty much field appearing there. I guess eventually we would want to be consistent across security solution and would have the same problem with value coming from different fields in the data

What about changing the logic on the flyout to the simpler one as well (either `observer.vendor` or pretty name from  dataset or just `-`) and have just one utility for this logic shared between table and flyout?

This difference in handling can be a constant source of nasty bugs in the future

Reply:
yep, sounds good. pushing a fix